cmd/skopeo/copy.go

```diff
@@ -63,8 +65,17 @@ func copyHandler(context *cli.Context) error {
 }
 
 var copyCmd = cli.Command{
-	Name:      "copy",
-	Usage:     "Copy an image from one location to another",
+	Name:  "copy",
+	Usage: "Copy an image (manifest, filesystem layers, signatures) from one location to another",
```

Comment:
Ah, I didn’t realize this comes from the existing man page.

Either way: I can’t quite see the utility of listing the individual components—but if we are listing the things to copy, then the list should be complete (i.e. also include `config.json`=“the image configuration" or whatever we want to call it). It seems simpler to me to drop the detailed list and to just say “copy an image” without listing the specific components.